getting-started/deployment_guide.md

```diff
@@ -24,6 +24,50 @@ curl -sfL http://get.kubearmor.io/ | sudo sh -s -- -b /usr/local/bin
 > [!NOTE] 
 > kArmor CLI provides a Developer Friendly way to interact with KubeArmor Telemetry. You can stream KubeArmor telemetry independently of kArmor CLI tool and integrate it with your chosen SIEM (Security Information and Event Management) solutions. [Here's a guide](https://github.com/kubearmor/kubearmor-relay-server/blob/main/README.md#streaming-kubearmor-telemetry-to-external-siem-tools) on how to achieve this integration. This guide assumes you have kArmor CLI to access KubeArmor Telemetry but you can view it on your SIEM tool once integrated.
 
+## KubeArmor EKS-plugin 
```

Comment:
Can we keep this heading entirely as a collapsible section under install. Next line to helm install.

Reply:
Created a collapsible section for Eks-plugin installation under install .